bigframes/ml/llm.py

```diff
@@ -154,8 +154,11 @@ def _create_bqml_model(self):
             )
 
         if self.model_name not in _TEXT_GENERATOR_ENDPOINTS:
-            raise ValueError(
-                f"Model name {self.model_name} is not supported. We only support {', '.join(_TEXT_GENERATOR_ENDPOINTS)}."
+            warnings.warn(
+                f"Model name '{self.model_name}' is not supported. "
+                f"We currently support the following models: {', '.join(_TEXT_GENERATOR_ENDPOINTS)}. "
+                "However, model names can change, and the supported models may be outdated. "
+                "You can try using this model name, but it might result in unexpected errors."
```

Comment:
Can we say "You can try using this model name, but it might result in unexpected errors." -> "You should use this model name only if you are sure that it is supported in BigQuery."

@GarrettWu

Reply:
Updated.